dpnp/dpnp_iface_searching.py

```diff
@@ -51,22 +49,38 @@
 __all__ = ["argmax", "argmin", "searchsorted", "where"]
 
 
-def argmax(x1, axis=None, out=None):
+def argmax(a, axis=None, out=None, *, keepdims=False):
```

Comment:
`dpnp.ndarray.argmin` and `dpnp.ndarray.argmax` need to be extended with `keepdims` keyword support